src/actions/guides/database/migrations.cr

```diff
@@ -155,9 +157,33 @@ class Guides::Database::Migrations < GuideAction
     ```crystal
     alter :users do
       add active : Bool, default: true, fill_existing_with: true
-      add otp_code : String, fill_existing_with: my_custom_otp_code_generator
+      add otp_code : String, fill_existing_with: "fake-otp-code-123"
     end
     ```
+    
+    > You can also use `fill_existing_with: :nothing` if your table is empty.
+    
+    If a static value will not work, try this:
+    
+    * If you have not yet released the app, consider using `fill_existing_with: :nothing`
+      and dropping the database `lucky db.drop` and recreating it with `lucky db.create && lucky db.migrate`.
+    * Consider making the type nilable `add otp_code : String?`, then fill the values with whatever value you need.
+      Then later make it required with `make_required :otp_code`
+    
+    ``` crystal
+    alter :users do
+      # Add nullable column first
+      add otp_code : String?
+    end
+    
+    # Then add values to it
+    UserQuery.new.each do |user|
+      User::BaseForm.udpate!(user, otp_code: CodeGenerator.generate)
+    end
+    
+    # Then make it non-nullable
+    make_required :otp_code
```

Comment:
Just to make this super crystal clear, can you put it all inside `def migrate`? Most the other migration examples do that as well. This is just to show that these aren't 3 code snippets used around the app somewhere, but literally all in the same method.

Reply:
Good point! Done!

And yeah there is `make_optional` and `make_required` :) I guess those aren't documented lol